### PR TITLE
Add some more libraries/runtimes

### PR DESCRIPTION
**Pull request by archeYR**
Added DirectX End-User Runtimes, Adobe Flash Player, JRE 6u45 and 8u11, Visual C++ 2017 Redistributable, XNA Framework 4.0 and also corrected typos in vc2010sp1run, vc2012run and vc2013run.
All these packages are working on ReactOS.

Please add Adobe Flash Player, JRE 6u45 and 8u11 to http://svn.reactos.org/packages
These can be obtained here: https://drive.google.com/open?id=1wKMiY0BSj_FiPrnV1h-08__JxcDxgaqB

### CODE REVIEW

File: fp.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = Adobe Flash Player 29
+Version = 29.0.0.140
+License = Freeware
+Description = This archive contains Flash Player and its plugins for Gecko and Chromium based browsers.
```
Comment (Mouvedia) [inline fp.txt]: The NPAPI version is compatible with Opera 12.

File: jre8.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[Section]
+Name = Java SE Runtime Environment 8
+Version = 8u11
+License = Oracle Binary Code License
+Description = Platform and libraries needed for running Java-based programs.
+Size = 29.7 MiB
+Category = 14
+URLDownload = http://svn.reactos.org/packages/jre-8u11-windows-i586.exe
```
Comment (sanchaez) [inline jre8.txt]: Java has an archive of these releases here: http://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase8-2177648.html

File: fp.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Category = 6
+URLSite = https://www.adobe.com/products/flashplayer.html
+URLDownload = http://svn.reactos.org/packages/fp_29.0.0.140.zip
+SHA1 = 1e969604941d21827cca70bda54319776493f4fa
+CDPath = none
+SizeBytes = 47169462
+
+[Section.0415]
+License = Darmowe
+Description = To archiwum zawiera program Flash Player i jego wtyczki do przeglądarek internetowych (NPAPI i PPAPI).
```
Comment (sanchaez) [inline fp.txt]: It's either PPAPI or NPAPI, not both.

File: fp.txt [hunk 4]
```diff
@@ -0,0 +1,17 @@
+[Section]
+Name = Adobe Flash Player 29
+Version = 29.0.0.140
+License = Freeware
+Description = This archive contains Flash Player and its plugins for web browsers (NPAPI and PPAPI).
+Size = 44.9 MiB
+Category = 6
+URLSite = https://www.adobe.com/products/flashplayer.html
+URLDownload = http://svn.reactos.org/packages/fp_29.0.0.140.zip
```
Comment (sanchaez) [inline fp.txt]: There is an official source for these packages - https://forums.adobe.com/thread/2464677

File: jre6.txt [hunk 5]
```diff
@@ -0,0 +1,41 @@
+[Section]
+Name = Java SE Runtime Environment 6
+Version = 6u45
+License = Oracle Binary Code License
+Description = Platform and libraries needed for running Java-based programs.
+Size = 16.2 MiB
+Category = 14
+URLDownload = http://svn.reactos.org/packages/jre-6u45-windows-i586.exe
```
Comment (sanchaez) [inline jre6.txt]: I'm not sure Oracle permits redistribution of these packages.

File: jre8.txt (hunk 2 above)
Comment (archeYR) [reply jre8.txt]: An Oracle account is needed to access these links.

File: fp.txt (hunk 3 above)
Comment (archeYR) [reply fp.txt]: My package has installers for both plugins.

File: fp.txt (hunk 4 above)
Comment (archeYR) [reply fp.txt]: All these links are leading to online installer, which does not work on ReactOS. And I would avoid using the online installer if possible anyway.

File: jre6.txt (hunk 5 above)
Comment (archeYR) [reply jre6.txt]: You already have a JRE 7u80 installer, so I assumed that you have an agreement/it's allowed to redistribute these packages.

**Comment by archeYR** [comment]
I managed to find links to the Flash Player plugins and JRE installers. I have split fp into the fpnpapi and fpppapi.

**Comment by learn-more** [comment]
Sorry for the delay!

These were all tested on ReactOS ?

**Comment by blackcrack** [comment]
@archeYR  well done :)
the Visual C++ 2017 did i also request in the past, but they want know my realname and the alias only be not allow.. hmm..

**Comment by archeYR** [comment]
virtualbox_reactos oldvbox_08_08_2018_20_25_57
Skyrim requires DirectX runtimes.

virtualbox_rostest_07_08_2018_23_15_38
Flash NPAPI plugin runs in Mypal 27.9.3.

virtualbox_rostest_07_08_2018_23_55_30
Flash PPAPI plugin runs in Opera 36 (without sound, but still might be useful).

virtualbox_reactos oldvbox_08_08_2018_20_35_57
MaryTTS 5.0 runs with Java Runtime Environment 6.

virtualbox_reactos oldvbox_08_08_2018_20_37_31
MaryTTS 5.2 runs with Java Runtime Environment 8.

virtualbox_reactos oldvbox_08_08_2018_20_53_22
Winevdm requires Vs C++ 2017 Redistributables (at least the commandline part works, sorry, couldn't find the better example).

virtualbox_reactos oldvbox_08_08_2018_12_24_03
Terraria requires XNA Framework 4.0 to run.

**Comment by learn-more** [comment]
Any objections against this getting merged?
Cc @JoachimHenze

**Pull request merged**